{
    "data_version": "4.0",
    "data_type": "CVE",
    "data_format": "MITRE",
    "CVE_data_meta": {
        "ID": "CVE-2022-0084",
        "ASSIGNER": "secalert@redhat.com",
        "STATE": "PUBLIC"
    },
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "A flaw was found in XNIO, specifically in the notifyReadClosed method. The issue revealed this method was logging a message to another expected end. This flaw allows an attacker to send flawed requests to a server, possibly causing log contention-related performance concerns or an unwanted disk fill-up."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "CWE-770 - Allocation of Resources Without Limits or Throttling",
                        "cweId": "CWE-770"
                    }
                ]
            }
        ]
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "n/a",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "xnio",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_affected": "=",
                                            "version_value": "Fixed in v3.x"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "references": {
        "reference_data": [
            {
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2064226",
                "refsource": "MISC",
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=2064226"
            },
            {
                "url": "https://access.redhat.com/security/cve/CVE-2022-0084",
                "refsource": "MISC",
                "name": "https://access.redhat.com/security/cve/CVE-2022-0084"
            },
            {
                "url": "https://github.com/xnio/xnio/pull/291",
                "refsource": "MISC",
                "name": "https://github.com/xnio/xnio/pull/291"
            },
            {
                "url": "https://github.com/xnio/xnio/commit/fdefb3b8b715d33387cadc4d48991fb1989b0c12",
                "refsource": "MISC",
                "name": "https://github.com/xnio/xnio/commit/fdefb3b8b715d33387cadc4d48991fb1989b0c12"
            }
        ]
    }
}